webpages/js/metaanalyses.js

```diff
@@ -1314,10 +1315,18 @@
     var addRowNode = _.findEl(table, 'tbody > tr.add');
 
     papers.forEach(function(paper, papIndex) {
+      var papTitleIndex = getFirstNonHiddenExp(paper);
```

Comment:
What we need to know in the loop below is when we are on the first visible experiment. We can use a boolean variable here, set it to true, and when we get past the isHiddenExp condition below, react to this variable, and then set it to false.

Then we can remove the getFirstNonHiddenExp function.